src/components/ImageGallery.jsx

```diff
@@ -762,14 +762,17 @@ class ImageGallery extends React.Component {
     return items.length >= 2;
   }
 
+  //SOLVED BY hunxjunedo : https://github.com/xiaolin/react-image-gallery/issues/767
+  //removed some minor errors, rmeove semi-colon
```

Comment:
You can remove the commented code, your contributions will be automatically registered based on your commit if this is merged.